pkg/config/control_plane.go

```diff
@@ -186,7 +186,9 @@ type ControlPlaneCache struct {
 }
 
 type ControlPlaneInsightCollector struct {
-	Schedule string `json:"schedule"`
+	Schedule          string `json:"schedule"`
+	RetryTime         int    `json:"retryTime"`
+	RetryIntervalHour int    `json:"retryIntervalHour"`
```

Comment:
Please set the default value for these fields. Currently, they are zero it means no jobs will be executed.

Reply:
OK 👍